.github/workflows/test-helm-chart.yaml

```diff
@@ -92,16 +86,16 @@ jobs:
           timeout: 900
           max-restarts: 15
 
-      - name: Setup for ingestion test
-        run: |
-          export WEB_POD=$(kubectl get pods -l role=web -o jsonpath="{.items[].metadata.name}")
-          kubectl exec $WEB_POD -- python manage.py setup_dev --no-data
+      - name: Setup PostHog for the ingestion test
+        run: ./ci/setup_ingestion_test.sh
 
-          echo "POSTHOG_WEB_HOSTNAME=$(kubectl get svc posthog-web -o jsonpath="{.spec.clusterIP}")" >> $GITHUB_ENV
-          echo "POSTHOG_EVENTS_HOSTNAME=$(kubectl get svc posthog-events -o jsonpath="{.spec.clusterIP}")" >> $GITHUB_ENV
+      - name: Fetch PostHog endpoints to use for the ingestion test
+        run: |
+          POSTHOG_API_ADDRESS=$(kubectl get svc -n posthog posthog-web -o jsonpath="{.spec.clusterIP}")
+          POSTHOG_EVENTS_ADDRESS=$(kubectl get svc -n posthog posthog-events -o jsonpath="{.spec.clusterIP}")
 
-          # Sleep until web service is ready to respond
-          sleep 20
```

Comment:
Will the endpoint be ready to respond immediately? I wouldn't expect helm upgrade to block until then.

Reply:
Okay, so the guarantee is that after `helm install` the site is immediately accessible? That's surprising for me and wouldn't work with e.g. DNS.

Reply:
The guarantee is that after helm install the resources are installed, this doesn't imply the site is immediately accessible tho (e.g. if the app takes 20 second to start). In this case anyway we only need to fetch the endpoint from the resources so we should be good.